fe/fe-core/src/test/java/org/apache/doris/statistics/StatisticsJobManagerTest.java

```diff
@@ -0,0 +1,178 @@
+package org.apache.doris.statistics;
+
+import mockit.Expectations;
```

Comment:
Pay attention to the import order. refer to https://doris.apache.org/zh-CN/developer-guide/fe-eclipse-dev.html#import-%E9%A1%BA%E5%BA%8F

Reply:
I remember that I have set it according to the documentation, but it doesn't seem to take effect for the test module, I will reset it~